js/ElementRendererProvider.js

```diff
@@ -219,6 +219,32 @@ export default class ElementRendererProvider {
     this.set(TYPES.LIST, (): HTMLElement => {
       const formEl = document.createElement('form');
       formEl.className = 'lp-json-pollock-layout lp-json-pollock-layout-form';
+      (formEl: any).afterRender = () => {
+        const allInputElArr = formEl.querySelectorAll('input');
+        if (allInputElArr.length > 0) {
+          for (let i = 0; i < allInputElArr.length; i += 1) {
+            const inputEl = allInputElArr[i];
+            if (inputEl.onclick) {
+              const funcToCall = inputEl.onclick;
+              inputEl.onclick = (event) => {
+                funcToCall.call(this, event, formEl);
+              };
+            }
+          }
+        }
+
+        // in form, the type button needs to be added so that the browser does not
+        // interpret button elements as submit button and trigger page refresh
+        const allBtnElArr = formEl.querySelectorAll('button');
```

Comment:
why not add the type on the `button` creation?

Reply:
I did this because I didn't want to change the behavior of existing button element. If I were to do this during `buttton` creation I could potentially mess up users who are already using this (i.e. custom styles that the brands have implemented on their end)

Reply:
I get it, you are right - we don't want to add any breaking change.
however, for the same reason - buttons on forms may look different than other buttons, not sure that this is the desired behavior.

a different approach may be to add it to the button creation and reset its styles on the head of [style.css](https://github.com/LivePersonInc/json-pollock/blob/master/scss/style.scss)

WDYT?